### PR TITLE
backend: Add optional tick_timeout argument to exchange functions

### PR DESCRIPTION
**Pull request by sgliner-ledger**

### CODE REVIEW

**Review by xchapron-ledger — COMMENTED** [review]
Maybe the default value can be changed?
Also can you bump the speculos required version?

**Comment by xchapron-ledger** [comment]
Can you bump the required speculos version?

File: pyproject.toml [hunk 1]
```diff
@@ -61,7 +61,7 @@ doc = [
     "docutils==0.16",
 ]
 speculos = [
-    "speculos>=0.5.0",
```
Comment (xchapron-ledger) [inline pyproject.toml:64]: Last speculos version is something like 0.7.1, why not using it?

Comment (sgliner-ledger) [reply pyproject.toml:64]: I can change it, but I was thinking the upgrade is not mandatory as it doesnt break compatibility

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**